pkg/clusterversion/cockroach_versions.go

```diff
@@ -471,6 +473,10 @@ var versionsSingleton = keyedVersions{
 		Key:     MarkerDataKeysRegistry,
 		Version: roachpb.Version{Major: 21, Minor: 1, Internal: 1164},
 	},
+	{
+		Key:     RegroleType,
+		Version: roachpb.Version{Major: 21, Minor: 1, Internal: 1166},
```

Comment:
let's just get additional sign-off to make sure it's ok to backport this. i believe it is... but if not then i guess the way to unblock the release would be to revert the regrole change.